spec/concurrent/struct_shared.rb

```diff
@@ -26,6 +26,18 @@
       expect(clazz.ancestors).to include described_class
     end
 
+    it 'ignores methods on ancestor classes' do
+      ancestor = described_class.ancestors.first
```

Comment:
`described_class.ancestors.first` is the same as `described_class`, no?
Unless prepended modules are involved here?

Reply:
Good catch! I double checked and it works as well using `ancestors.last` 👍🏻